.changeset/large-guests-fix.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/adapter-cloudflare-workers': minor
+---
+
+fix: throws KVError when non-existing asset
```

Comment:
this will show up in the changelog, so should be worded in a way that describes the new behavior rather than just the old behavior
```suggestion
fix: return 404 for non-existent asset rather than throwing KVError
```